src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -243,6 +252,10 @@ export class PostItCollectionsComponent implements OnInit, AfterViewInit, OnDest
       }, {});
     });
     this.subscriptions.add(collectionsSubscription);
+
+    const loadedSubscription = this.store.select(selectCollectionsLoaded)
+      .subscribe(loaded => this.collectionsLoaded = loaded);
+    this.subscriptions.add(loadedSubscription);
```

Comment:
Maybe we would need similar mechanism in other components. Then it would make more sense to implement it as another `CollectionsState` property (e.g. `loading:boolean`) which will have its own action to change it. This action will be dispatched with value `true` right before a backend call is made in `CollectionsEffects` and with `false` after `GetSuccess` action.

Reply:
I just need to know when collections are loaded. The function above has only a bad name..

Reply:
Oh, I see. It's something different from what I meant.